gobblin-runtime/src/main/java/gobblin/scheduler/PathAlterationListenerAdaptorForMonitor.java

```diff
@@ -0,0 +1,209 @@
+package gobblin.scheduler;
+
+import java.io.IOException;
+import java.util.Properties;
+
+import org.apache.commons.configuration.ConfigurationException;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileStatus;
+import org.apache.hadoop.fs.Path;
+import org.slf4j.LoggerFactory;
+import org.slf4j.Logger;
+
+import gobblin.configuration.ConfigurationKeys;
+import gobblin.runtime.JobException;
+import gobblin.runtime.listeners.EmailNotificationJobListener;
+import gobblin.runtime.listeners.RunOnceJobListener;
+import gobblin.util.SchedulerUtils;
+
+
+
+/**
+ * Inner subclass of PathAlterationListenerAdaptor for implementation of Listen's methods,
+ * avoiding anonymous class
+ */
+public class PathAlterationListenerAdaptorForMonitor extends gobblin.util.filesystem.PathAlterationListenerAdaptor {
```

Comment:
Please import `gobblin.util.filesystem.PathAlterationListenerAdaptor` instead of using the fully qualified class.